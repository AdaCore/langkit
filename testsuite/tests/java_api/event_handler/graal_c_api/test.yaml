driver: python
sync_trees:
    - ../common
control:
    - [SKIP, "not has_java", "Java tests disabled"]
    - [SKIP, "not has_native_image", "Native-image tests disabled"]
