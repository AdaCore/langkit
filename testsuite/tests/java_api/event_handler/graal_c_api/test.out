=== Non null callbacks ===
--- Unit parsed callback
unit: <AnalysisUnit "example">
reparsed: false

--- Unit parsed callback
unit: <AnalysisUnit "example">
reparsed: true

--- Unit requested callback
name: foo_1
from: <AnalysisUnit "example">
found: false
is_not_found_error: false

--- Unit requested callback
name: foo_2
from: <AnalysisUnit "example">
found: true
is_not_found_error: false

--- Unit requested callback
name: foo_3
from: <AnalysisUnit "example">
found: true
is_not_found_error: true

--- Unit requested callback
name: foo_4
from: <AnalysisUnit "example">
found: false
is_not_found_error: true

=== Null callbacks ===
Done
